src/components/TextField/TextField.tsx

```diff
@@ -43,8 +83,10 @@ export const TextField: React.FC<Props> = ({
           type="text"
           placeholder={`Enter ${label}`}
           value={value}
-          onChange={event => onChange(event.target.value)}
-          onBlur={() => setToched(true)}
+          onChange={handleInputChange}
+          onBlur={() => {
+            blurring();
+          }}
```

Comment:
Use more meaningful names
```suggestion
          onBlur={onBlur}
```